### PR TITLE
add toggleterm highlights and initial bufferline highlights

### LINKED ISSUE #76: Support toggleterm highlights config
A recent patch to toggleterm made it so that, without any custom config, the background would always be dark even if neovim started up with `bg=light` and rose-pine correctly auto went into the light alt.

I was able to work around the problem with the help of the toggleterm author https://github.com/akinsho/toggleterm.nvim/pull/166#issuecomment-1098513458

However, I don't really understand vim theming. I'm happy to keep my manual fixes for toggle term. But also I know rose-pine sets some themes for other plugins. Should (or even can?) rose-pine come with specific settings for toggleterm?

### PR DESCRIPTION
**Pull request by mvllow**
closes #76

### CODE REVIEW

**Comment by mvllow** [comment]
This adds a starting point for supporting plugins in which the user sets their own highlights. Included are bufferline and toggleterm:

```lua
use({
	'akinsho/bufferline.nvim',
	event = 'ColorScheme',
	config = function()
		local highlights = require('rose-pine.plugins.bufferline')
		require('bufferline').setup({
			highlights = highlights,
		})
	end,
})
use({
	'akinsho/toggleterm.nvim',
	event = 'ColorScheme',
	config = function()
		local highlights = require('rose-pine.plugins.toggleterm')
		require('toggleterm').setup({
			highlights = highlights,
		})
	end,
})
```

> Notice that `event = 'ColorScheme'` is necessary when Rosé Pine uses the dawn variant.

For anyone wanting to contribute, please reach out here. Ideally we would have a PR per plugin – simply use this PR for reference 😌

**Comment by will** [comment]
Thanks!

File: readme.md [hunk 1]
```diff
@@ -66,6 +66,35 @@ use({
...
+		local highlights = require('rose-pine.plugins.bufferline')
+		require('bufferline').setup({ highlights = highlights })
+	end
+})
+```
+
+- [toggleterm.nvim](https://github.com/akinsho/toggleterm.nvim)
+
+```lua
+require('toggleterm').setup({ highlights = highlights })
```
Comment (will) [inline readme.md:87]:
```suggestion
```

This line I think was probably a leftover or like copy-and-paste error?

Comment (mvllow) [reply readme.md:87]: Good catch, thanks!

**Pull request merged**